ember-accounts/app/mixins/analytics.js

```diff
@@ -0,0 +1,15 @@
+import Ember from 'ember';
+
+//A temporary solution to get GA shipped, Really what I want here
+//is conventionally queued Analytics events - @discorick
+export default Ember.Mixin.create({
+  actions: {
+    purchase(model) {
+      window.analytics.page({
+        url: `/settings/${this.get('model.login')}/upgrade`,
```

Comment:
Confirmed. Guess it doesn't matter. :grinning: 

Next question: thoughts on setting `referrer` here to `window.location.toString()`?